src/components/__tests__/search-bar-test.jsx

```diff
@@ -16,20 +16,40 @@ describe('search-bar', () => {
   });
 
   it('Handles search queries with a protocol correctly', () => {
-    const searchBar = shallow(<SearchBar onSearch={onSearch} />);
+    const onSearch = jest.fn();
+    const searchBar = shallow(<SearchBar onSearch={onSearch} startDate={null} endDate={null} />);
```

Comment:
`startDate` and `endDate` don’t look like they are valid props on `SearchBar` (they *are* definitely on `SearchDatePicker`). Is this right?

Reply:
Bad copy/paste, my bad.